### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by fededirocco**
## Summary
* Implemented footer
* Added scss for footer

## Trello Card
https://trello.com/c/mNNk5bAb/14-footer

## Screenshots
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -90,5 +90,10 @@ <h1 class="name">Django</h1>
       </div>      
     </div>
   </div>
+  <div class="footer">
+    <h1 class="text">WE ARE GAME CHANGERS</h1>
```
Comment (jjuli93) [inline index.html]: Use css to style for the uppercase

File: scss/footer.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.footer {
+    align-items: center;
+    background-color: $blue;
+    bottom: 0;
```
Comment (jjuli93) [inline scss/footer.scss]: why?

File: scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    display: flex;
+    flex-direction: column;
+    left: 0;
```
Comment (jjuli93) [inline scss/footer.scss]: why?

File: index.html [hunk 4]
```diff
@@ -90,5 +90,10 @@ <h1 class="name">Django</h1>
...
+    <img class="logo" src="img/wolox_logo_white.svg" alt="Wolox logo" />
+    <p class="copyright">Copyright 2018. Wolox. All rights reserved</p>
```
Comment (jjuli93) [inline index.html]: https://stackoverflow.com/questions/1908234/when-to-use-span-instead-p

File: index.html [hunk 5]
```diff
@@ -90,5 +90,10 @@ <h1 class="name">Django</h1>
       </div>      
     </div>
   </div>
+  <div class="footer">
```
Comment (FrankIglesias) [inline index.html]: You can use a more semantic tag like `<footer>`

File: scss/footer.scss (hunk 2 above)
Comment (fededirocco) [reply scss/footer.scss]: Yes, its wrong

File: scss/footer.scss (hunk 3 above)
Comment (fededirocco) [reply scss/footer.scss]: Idem bottom 0, wrong

File: scss/footer.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
+footer {
```
Comment (FrankIglesias) [inline scss/footer.scss]: check indentation

File: scss/footer.scss [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    background-color: $blue;
+    display: flex;
+    flex-direction: column;
+    padding: $padding-large;
+    width: 100%;
+
+    .text {
+        color: $white;
+        font-size: 23px;
+        padding: $padding-small 0 $padding-small;
```
Comment (FrankIglesias) [inline scss/footer.scss]: replace this for `padding: $padding-small 0;`

**Pull request closed without merging**